app/Http/Controllers/api/v1/ApplicationController.php

```diff
@@ -76,6 +77,25 @@ public function updateSettings(UpdateSetting $request)
             return $setting !== null;
         }));
 
+        // reset record_attendance for running meetings and remove attendance data,
+        // as new attendees would not see a warning, but are recorded
+        // if this global setting is re-enabled during the meeting
+        if (setting('attendance.enabled') && !$request->attendance['enabled']) {
+            $meetings = Meeting::whereNull('end')->where('record_attendance', '=', 1)->get();
```

Comment:
What if a meeting already ended? Shouldn't the recorded attendees be deleted?

Reply:
In case the meetings ended, all attendees have agreed to be logged before joing the meeting. We can keep the data for the retention period, in case this setting is changed later